### ISSUE TITLE
Tests are triggering new swarmplot warning

### ISSUE THREAD

**Issue by mwaskom** [mwaskom/seaborn#2053]
```
=============================== warnings summary ===============================
seaborn/categorical.py::seaborn.categorical.swarmplot
  /Users/mwaskom/code/seaborn/seaborn/categorical.py:1325: UserWarning: 6.5% of the points cannot be placed; you may want to decrease the size of the markers or use stripplot.
    warnings.warn(msg, UserWarning)

seaborn/categorical.py::seaborn.categorical.swarmplot
  /Users/mwaskom/code/seaborn/seaborn/categorical.py:1325: UserWarning: 2.4% of the points cannot be placed; you may want to decrease the size of the markers or use stripplot.
    warnings.warn(msg, UserWarning)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
```

**Comment by mwaskom** [comment]
Several of the examples in the docs, too.

**Comment by mwaskom** [comment]
Closed by #2216
